__tests__/Auth0Client.test.ts

```diff
@@ -890,7 +1261,7 @@ describe('Auth0Client', () => {
       const auth0 = setup({
         legacySameSiteCookie: false
       });
-      await auth0.checkSession();
+      await checkSession(auth0);
```

Comment:
These two `checkSession` relied on other tests executed first and setting up a mock, so they would fail when ran in isolation.

I updated these two tests to use the same pattern as we have with the others, where we decorate the call by wrapping it and setting up the required mocking in the wrapper.